lib/tasks/expire_consultations.rake

```diff
@@ -0,0 +1,6 @@
+namespace :expire do
+  desc "Update consultation status"
+  task consultations: :environment do
+    Consultation.where("status != ? AND response_deadline < ?", Consultation.statuses['expired'], Date.today).update(status: :expired)
```

Comment:
```suggestion
    Consultation.where("status != ? AND response_deadline < ?", Consultation.statuses['expired'], Date.yesterday).update(status: :expired)
```

Since we are doing it at 12:00AM, we should use yesterday.